### DISCUSSION TITLE
Ideas for futures

### DISCUSSION THREAD

**Discussion by new92** [new92/igfi#11] [Ideas]
If you have any ideas for how IGFI can be improved or if you have general suggestions for features you'd like to see in IGFI, please feel free to share them here. Your ideas are highly valued! 👍 Please describe your suggestions in as much detail as possible. ✔️

**Comment by brxndonnn** [comment]
so its a pretty basic idea but how about a likes increaser, for example i have 127 likes on a post but i want it at lets say 300...is that possible?
